app/presenters/summary/sections/other_outgoings_details.rb

```diff
@@ -1,21 +1,31 @@
 module Summary
   module Sections
     class OtherOutgoingsDetails < Sections::BaseSection
+      include TypeOfMeansAssessment
+
       def show?
         outgoings.present? && super
       end
 
-      # rubocop:disable Metrics/MethodLength
+      # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
       def answers
-        [
+        answers = [
           Components::ValueAnswer.new(
             :income_tax_rate_above_threshold, outgoings.income_tax_rate_above_threshold,
             change_path: edit_steps_outgoings_income_tax_rate_path
           ),
-          Components::ValueAnswer.new(
-            :partner_income_tax_rate_above_threshold, outgoings.partner_income_tax_rate_above_threshold,
-            change_path: edit_steps_outgoings_partner_income_tax_rate_path
-          ),
+        ]
+
+        if FeatureFlags.partner_journey.enabled? && include_partner_in_means_assessment?
```

Comment:
I'd prefer if this was based on whether the attribute `partner_income_tax_rate_above_threshold` is present rather than if the partner is included in the assessment

Reply:
Hmm hmm, just wondering whether what I've got will cover off if they answer it then go back and change the partner status? I'm not super bothered either way so long as the partner means are reset effectively

Reply:
True, I don't think we are resetting these attributes actually 😅 ok good with me!

Reply:
TY! Can take another look later